### PR TITLE
Infra Observability 

### PR DESCRIPTION
**Pull request by mrhoads**
This PR adds screenshots and update info on infra observability and attempts to incorporate feedback from PR #40 (since closed).

### CODE REVIEW

File: docs/azure_jumpstart_ag/contoso_hypermarket/observability/infrastructure/_index.md [hunk 1]
```diff
@@ -5,4 +5,90 @@ title: Kubernetes observability
...
+- Third, the combination of Prometheus and Grafana aids in troubleshooting and root cause analysis.
+
+## Architecture
+
+The architecture of the observability infrastructure stack uses the [Kube Prometheus Stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack). This stack is comprised of Kubernetes manifests, Grafana dashboards, and Prometheus rules, which are easily deployed and provide end-to-end monitoring of Kubernetes clusters.
+
+In the store locations, Prometheus instances are deployed to periodically collect metrics from the K3s cluster. A separate Grafana instance is set up to centralize monitoring and visualization. The Prometheus instances send metrics data to this central Grafana instance. Grafana dashboards are then configured to display relevant metrics, enabling operators and administrators to monitor the health and performance of the entire infrastructure.
+
```
Comment (sebassem) [inline docs/azure_jumpstart_ag/contoso_hypermarket/observability/infrastructure/_index.md:27]: I think we are missing the diagram here

File: docs/azure_jumpstart_ag/contoso_hypermarket/observability/infrastructure/_index.md [hunk 2]
```diff
@@ -5,4 +5,90 @@ title: Kubernetes observability
...
+## Grafana dashboards
+
+Grafana's dashboards in Contoso's implementation provide a visually appealing and customizable interface for monitoring their infrastructure. With Grafana, they can create intuitive and interactive dashboards that display key metrics and insights, empowering them to make data-driven decisions and quickly identify any issues or trends within their cloud to edge infrastructure. The following Grafana dashboards are automatically deployed as part of advanced automation for all the manufacturing plants:
+
+- **Kubernetes / Views / Global**: The Kubernetes Global Dashboard in Grafana offers a concise overview of your Kubernetes cluster, allowing you to quickly assess its overall health and performance. The dashboard includes panels that highlight key metrics, such as total cluster CPU, RAM, and network utilization, as well as resource usage across namespaces and nodes. Additionally, it tracks the number of resource types used in the cluster and helps detect misconfigured application resources by comparing real usage with requested and limited resources.
```
Comment (sebassem) [inline docs/azure_jumpstart_ag/contoso_hypermarket/observability/infrastructure/_index.md:32]:
```suggestion
- **Kubernetes / Views / Global**: The Kubernetes Global Dashboard in Grafana offers a concise overview of Contoso's Kubernetes cluster, allowing them to quickly assess its overall health and performance. The dashboard includes panels that highlight key metrics, such as total cluster CPU, RAM, and network utilization, as well as resource usage across namespaces and nodes. Additionally, it tracks the number of resource types used in the cluster and helps detect misconfigured application resources by comparing real usage with requested and limited resources.
```

File: docs/azure_jumpstart_ag/contoso_hypermarket/observability/infrastructure/_index.md [hunk 3]
```diff
@@ -5,4 +5,90 @@ title: Kubernetes observability
...
+
+- **Node Exporter Full**: The Kubernetes Nodes Dashboard in Grafana provides a detailed view of node-level metrics and resources in your Kubernetes cluster. It enables you to monitor CPU and RAM usage, track pods running on each node, and identify any resource anomalies or performance issues. The dashboard also offers system-level metrics such as system load, context switches, and file descriptors, allowing for troubleshooting and optimization. Additionally, it provides insights into storage capacity, volumes, and I/O operations on the nodes, aiding in the effective management of storage resources.
```
Comment (sebassem) [inline docs/azure_jumpstart_ag/contoso_hypermarket/observability/infrastructure/_index.md:34]:
```suggestion
- **Node Exporter Full**: The Kubernetes Nodes Dashboard in Grafana provides a detailed view of node-level metrics and resources in Contoso's Kubernetes cluster. It enables you to monitor CPU and RAM usage, track pods running on each node, and identify any resource anomalies or performance issues. The dashboard also offers system-level metrics such as system load, context switches, and file descriptors, allowing for troubleshooting and optimization. Additionally, it provides insights into storage capacity, volumes, and I/O operations on the nodes, aiding in the effective management of storage resources.
```

File: docs/azure_jumpstart_ag/contoso_hypermarket/observability/infrastructure/_index.md [hunk 4]
```diff
@@ -5,4 +5,90 @@ title: Kubernetes observability
...
+> The Kubernetes dashboards included in this guide are based on the work of publicly available dashboards. We would like to express our appreciation for authors' efforts in creating this insightful dashboard that enhances our monitoring capabilities for Kubernetes.
+> The original dashboards can be found at:
+> - [Node Exporter Full](https://grafana.com/grafana/dashboards/1860-node-exporter-full)
+> - [Grafana Dashboards Kubernetes](https://github.com/dotdc/grafana-dashboards-kubernetes)
+
+## Accessing dashboards
+
+To view the health of the Kubernetes clusters, access the Grafana dashboards by doing the following:
+
+- Connect to the Client VM _Ag-VM-Client_ using the instruction in the [deployment guide](../deployment/)
```
Comment (sebassem) [inline docs/azure_jumpstart_ag/contoso_hypermarket/observability/infrastructure/_index.md:46]:
```suggestion
- Connect to the Client VM _Ag-VM-Client_ using the instructions in the [deployment guide](../deployment/)
```

**Review by sebassem — CHANGES_REQUESTED** [review]
LGTM, just minor updates and missing diagram

**Comment by mrhoads** [comment]
closing after fetching latest updates from canary

**Pull request closed without merging**